src/hooks/__tests__/usePrefersColorScheme.test.ts

```diff
@@ -0,0 +1,13 @@
+import usePrefersColorScheme from '@/hooks/usePrefersColorScheme'
+import { renderHook } from '@/tests/test-utils'
+
+describe('usePrefersColorScheme', () => {
+  it('should return the initial color scheme', () => {
```

Comment:
Could we add more tests for the listeners of the media query which would set the result to dark / light?

Reply:
Added. I found it a bit difficult to mock the media query event without adding the dependency. Maybe we can look at it quickly together

Reply:
I can have a look later to do it as described here:
https://jestjs.io/docs/26.x/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom